### PR TITLE
Implement Jwt extended Refresh token for user authentication, Added C…

### PR DESCRIPTION
**Pull request by Hammad047**
This PR contain config file and separated all directories followed by pep8 format

### CODE REVIEW

File: app/api.py [hunk 1]
```diff
@@ -21,13 +24,12 @@ def signup():
         data = request.get_json()  # getting data from POSTMAN
         name = data['name']  # get name
         email = data['email']  # get email
+        print('This is my email: ', email)
```
Comment (maimoonaiqbal94) [inline app/api.py]: remove print statements and dummy code.

File: app/api.py [hunk 2]
```diff
@@ -38,23 +40,28 @@ def signin():
         data = request.get_json()  # get data from postman
         email = data['email']  # get email
         password = data['password']  # get password
-        userdata = db.session.query(user_data).filter_by(
+        userdata = db.session.query(users.Users).filter_by(
```
Comment (maimoonaiqbal94) [inline app/api.py]: model name should be defined as a singular entity.. i.e. User instead of Users

File: app/api.py [hunk 3]
```diff
@@ -71,35 +78,49 @@ def createdataset():
         requests.get(url, headers=headers).content.decode('utf-8'))
     for data in response['results']:
         print(f'data fetched: {data}')
-        check = db.session.query(car_data).filter_by(object_id=data['objectId']).first()
+        check = db.session.query(cars.Cars).filter_by(object_id=data['objectId']).first()
```
Comment (maimoonaiqbal94) [inline app/api.py]: same case here. Car instead of Cars

File: authentication/email.py [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import smtplib
+from authentication import *
+
+def send_email(self, email_receiver):
+    email_sender = 'hammadkhanniazi420@yahoo.com'  # enter your email
+    email_sender_password = 'rdhgdzlwxnkwyqrv'  # enter your password (app-password)
+    subject = 'Email Verification'
```
Comment (maimoonaiqbal94) [inline authentication/email.py]: create a file consts, where you can define these constants and import from there.

Comment (Hammad047) [reply authentication/email.py]: Added

File: app/api.py (hunk 3 above)
Comment (Hammad047) [reply app/api.py]: Changed

File: app/api.py (hunk 2 above)
Comment (Hammad047) [reply app/api.py]: Changed

File: app/api.py (hunk 1 above)
Comment (Hammad047) [reply app/api.py]: Removed

File: app/api.py [hunk 5]
```diff
@@ -1,60 +1,86 @@
...
+from models import user, car
+from authentication.email import *
+from werkzeug.security import check_password_hash
+from flask_jwt_extended import create_access_token
+from flask_jwt_extended import create_refresh_token
+from flask_jwt_extended import get_jwt_identity
+from flask_jwt_extended import jwt_required
+from flask_jwt_extended import JWTManager
+
+jwt = JWTManager(app)
```
Comment (maimoonaiqbal94) [inline app/api.py]: can you create this globally,, as you have created DB?

Comment (Hammad047) [reply app/api.py]: Sure, let me check it out.

Comment (maimoonaiqbal94) [reply app/api.py]: is this fixed?

Comment (Hammad047) [reply app/api.py]: Yes, I fixed it.

**Pull request merged**